rdt/performance/performance.py

```diff
@@ -57,6 +63,47 @@ def evaluate_transformer_performance(transformer, dataset_generator):
             transform_size=transform_size,
         )
         size = np.array([fit_size, transform_size, transform_size] * 2)
-        out.append(performance / size)
+        performance = performance / size
+        if verbose:
+            performance['Number of fit rows'] = fit_size
+            performance['Number of transform rows'] = transform_size
+            performance['Dataset'] = dataset_generator.__name__
+            performance['Transformer'] = f'{transformer.__module__ }.{transformer.__name__}'
 
-    return pd.DataFrame(out).max(axis=0)
+        out.append(performance)
+
+    summary = pd.DataFrame(out)
+    if verbose:
+        return summary
+
+    return summary.max(axis=0)
+
+
+def evaluate_transformers(transformers=None):
+    """Evaluate the performance for each transformer against all applicable datasets.
+
+    Args:
+        transformers (list<rdt.transformers.BaseTransformer>):
+            List of transformer classes to evaluate.
+
+    Returns:
+        pandas.DataFrame:
+            A table containing the times and memory usages for ``fit``, ``transform`` and
+            ``reverse_transform`` for each transformer provided against every dataset generator
+            that applies to it. The table also contains the latest commit number.
```

Comment:
What do you mean by `The table contains the latest commit number.`?

Reply:
good catch I need to remove this